.github/workflows/audit.yml

```diff
@@ -19,6 +23,7 @@ jobs:
       contents: read
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/audit-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: cargo audit
+        run: |
+          cargo audit
```

Comment:
this seems to be missing cargo install?

Reply:
See https://github.com/actions/runner-images/blob/ubuntu22/20230129.2/images/linux/Ubuntu2204-Readme.md#rust-tools